GPU/Directx9/TextureCacheDX9.cpp

```diff
@@ -1012,7 +1011,7 @@ void TextureCacheDX9::ApplyTextureFramebuffer(TexCacheEntry *entry, VirtualFrame
 			// And also the UVs, same order.
 			const float uvleft = u1 * invWidth;
 			const float uvright = u2 * invWidth;
-			const float uvtop = 1.0f - v1 * invHeight;
+			const float uvtop = 1.0f - v1 * invHeight;  // TODO: Seems we should ditch the "1.0f - "
```

Comment:
Yeah, definitely, the positions need to agree too.  Otherwise it won't use the correct subset.

-[Unknown]